config.cc

```diff
@@ -52,6 +53,12 @@ namespace
       result = QDir::fromNativeSeparators( QString::fromWCharArray( _wgetenv( L"APPDATA" ) ) );
     #else
       char const * pathInHome = ".goldendict";
+      #if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
+        if (!result.exists(pathInHome)) {
+          result = QDir(QStandardPaths::locate(QStandardPaths::ConfigLocation, QString(), QStandardPaths::LocateDirectory));
+          pathInHome = "goldendict";
+        }
+      #endif
```

Comment:
Please reformat the added code in GoldenDict's code style (braces, parentheses).